fe/fe-core/src/main/java/com/starrocks/catalog/PhysicalPartitionImpl.java

```diff
@@ -159,12 +162,27 @@ public long getParentId() {
     public void setParentId(long parentId) {
         this.parentId = parentId;
     }
- 
+
+    // DEPRECATED
     @Override
     public long getShardGroupId() {
```

Comment:
what about old table created before this pr, partition <-> shard group is changed?

Reply:
The shard_group_id in partition of old table will be assigned to baseIndex when replay